src/apps/lwaftr/lwdebug.lua

```diff
@@ -1,20 +1,20 @@
 module(..., package.seeall)
 
 local bit = require("bit")
-local band, bor, rshift = bit.band, bit.bor, bit.rshift
+local band, rshift = bit.band, bit.rshift
 
 function pp(t) for k,v in pairs(t) do print(k,v) end end
 
 function print_ethernet(addr)
-   chunks = {}
+   local chunks = {}
```

Comment:
OK, I think you're right. I created an issue to carry out this change in a different PR https://github.com/Igalia/snabb/issues/485